Riot/Modules/Settings/SettingsViewController.m

```diff
@@ -1490,7 +1323,7 @@ - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(N
             
             cell = surnameCell;
         }
-        else if (userSettingsEmailStartIndex <= row &&  row < userSettingsNewEmailIndex)
+        else if (userSettingsEmailStartIndex == row)
```

Comment:
I would consider all the potential linked emails, even if only one is expected FTM
`else if (userSettingsEmailStartIndex <= row &&  row < userSettingsPhoneStartIndex)`